chatterbot/corpus/data/english/policies.corpus.json

```diff
@@ -0,0 +1,4 @@
+{
+    "policies": [
+    ]
+}
```

Comment:
This is just a blank file and doesn't contribute any knowledge to the dialog corpus.